dnf.spec

```diff
@@ -174,6 +174,9 @@ mv $RPM_BUILD_ROOT%{_bindir}/dnf-automatic-2 $RPM_BUILD_ROOT%{_bindir}/dnf-autom
 rm $RPM_BUILD_ROOT%{_bindir}/dnf-automatic-3
 %endif
 
+# This will eventually be the new default location for repo files
+mkdir $RPM_BUILD_ROOT%{_sysconfdir}/distro.repos.d
```

Comment:
@jsilhan Would `%ghost` allow it to recursively delete data it doesn't own if DNF were to be uninstalled?